cmd/loadtest/loadtest.go

```diff
@@ -1486,9 +1521,9 @@ func loadtestAvailStore(ctx context.Context, c *gsrpc.SubstrateAPI, nonce uint64
 	return
 }
 
-func configureTransactOpts(tops *bind.TransactOpts) *bind.TransactOpts {
+func configureTransactOpts(tops *bind.TransactOpts, legacy bool) *bind.TransactOpts {
```

Comment:
Could you read `legacy` from `ltp.LegacyTransactionMode` rather than adding a boolean flag and refactoring all of the functions calls?